docs/device_api.md

```diff
@@ -151,6 +152,7 @@ Fiware-ServicePath: /
 		"device_id": "[DEV_ID]",
 		"entity_name": "[ENTITY_ID]",
 		"entity_type": "thing",
+		"protocol":"IoTA-UL",
```

Comment:
I understand that IOTAManager (which at the end is the entry point for IOTAs) is able to process the protocol when it comes in the payload instead of as URI param. Is correct?

Reply:
Checked at skype with @manucarrace. It's ok. NTC